evap/evaluation/management/commands/scss.py

```diff
@@ -0,0 +1,39 @@
+import os
+import subprocess # nosec
+
+from django.core.management.base import BaseCommand
+from django.conf import settings
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--watch', action='store_true',
+            help='Watch stylesheets and recompile when they change.',
+        )
+        parser.add_argument(
+            '--release', action='store_true',
+            help='Compress output stylesheet and do not generate source maps.'
+                 ' Indented to use in production deployment.',
```

Comment:
Indented ;)

Reply:
That’s a tricky flip-flop of characters ^^